src/providers/VoiceFocusProvider/index.tsx

```diff
@@ -32,14 +32,20 @@ interface VoiceFocusState {
 
 const VoiceFocusContext = createContext<VoiceFocusState | null>(null);
 
-const VoiceFocusProvider: React.FC<Props> = ({ spec, options, children }) => {
+const VoiceFocusProvider: React.FC<Props> = ({ 
+  spec, 
+  options, 
+  createMeetingResponse,
```

Comment:
Will the VF provider now accepting `createMeetingResponse` can be re-used with multiple meetings? We separated out the VF provider because this can used for multiple meetings, can you once check with Richard on this, tagging @jing-chen1 for better context on this.

Reply:
@richnew10 does default voice focus support multiple meetings? how does that work? as far as I know, there was no mixing logic for multiple microphone streams.

Reply:
It's separate because:

1. You typically want to initialize VF prior to joining a meeting, and perhaps even before knowing which meeting you're joining. With echo reduction you can no longer do the latter, but you can do the former (just grab the `CreateMeetingResponse` from the server first — it's not the same as joining).
2. You might want to use the same microphone input as input to multiple meetings without running noise suppression multiple times. E.g., COVID testing supervision where a doctor will read instructions to six participants in six 1:1 meetings. This will work with echo reduction: request ER for all meetings, supply the `CreateMeetingResponse` of any of them when creating the device, and [observe the meeting audio for all meetings](https://github.com/aws/amazon-chime-sdk-js/blob/master/guides/09_Amazon_Voice_Focus.md#2-enabling-echo-reduction-at-the-client-level), which will use all of the output streams mixed together as the echo signal.

Reply:
Thanks for explanation @richnew10 ! We just have to test and document the behavior for multiple meetings to help builders understand it better. Took some time to imagine the scenario here :)